companion/manifest.json

```diff
@@ -2,8 +2,8 @@
 	"id": "christie-spyder",
 	"name": "christie-spyder",
 	"shortname": "spyder",
-	"description": "spyder plugin for Companion",
-	"version": "0.0.0",
+	"description": "Spyder plugin for Companion",
+	"version": "2.4.0",
```

Comment:
This should be left at '0.0.0'.
The build process migrates the version from `package.json`